src/domain/constants.js

```diff
@@ -155,6 +155,8 @@ export const HORIZONTAL_ALIGNMENT = {
   right: 'right'
 };
 
+export const CDN_URL_PREFIX = 'cdn://';
```

Comment:
This could also be used in `storage-service` and `storage-utils`.

Reply:
Nice, thanks for spotting those. Used there too.

Reply:
:+1: